docker-compose.yml

```diff
@@ -0,0 +1,219 @@
+      KAFKA_REST_HOST_NAME: rest-proxy-1
+      KAFKA_GROUP_MIN_SESSION_TIMEOUT_MS: 5000
+      KAFKA_AUTO_COMMIT_INTERVAL_MS: 300
+      KAFKA_REST_COMPRESSION_TYPE: lz4
+
+  #---------------------------------------------------------------------------#
+  # RADAR JDBC connector                                                     #
+  #---------------------------------------------------------------------------#
+  radar-jdbc-connector:
+    image: mpgxvii/radar-jdbc-connector:latest
+    restart: on-failure
+    volumes:
+      - ./docker/sink-timescale.properties:/etc/kafka-connect/sink-timescale.properties
+    depends_on:
+      - zookeeper-1
+      - zookeeper-2
+      - zookeeper-3
+      - kafka-1
+      - kafka-2
+      - kafka-3
+      - schema-registry-1
+      - timescaledb
+    environment:
+      CONNECT_BOOTSTRAP_SERVERS: PLAINTEXT://kafka-1:9092,PLAINTEXT://kafka-2:9092,PLAINTEXT://kafka-3:9092
+      CONNECT_REST_PORT: 8083
+      CONNECT_GROUP_ID: 'default'
+      CONNECT_CONFIG_STORAGE_TOPIC: 'default.config'
+      CONNECT_OFFSET_STORAGE_TOPIC: 'default.offsets'
+      CONNECT_STATUS_STORAGE_TOPIC: 'default.status'
+      CONNECT_KEY_CONVERTER: 'io.confluent.connect.avro.AvroConverter'
+      CONNECT_VALUE_CONVERTER: 'io.confluent.connect.avro.AvroConverter'
+      CONNECT_KEY_CONVERTER_SCHEMA_REGISTRY_URL: 'http://schema-registry-1:8081'
+      CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL: 'http://schema-registry-1:8081'
+      CONNECT_INTERNAL_KEY_CONVERTER: 'org.apache.kafka.connect.json.JsonConverter'
+      CONNECT_INTERNAL_VALUE_CONVERTER: 'org.apache.kafka.connect.json.JsonConverter'
+      CONNECT_OFFSET_STORAGE_FILE_FILENAME: '/var/lib/kafka-connect-jdbc/logs/connect.offsets'
+      CONNECT_REST_ADVERTISED_HOST_NAME: 'radar-jdbc-connector'
+      CONNECT_ZOOKEEPER_CONNECT: zookeeper-1:2181,zookeeper-2:2181,zookeeper-3:2181
+      CONNECTOR_PROPERTY_FILE_PREFIX: 'sink-timescale'
+      KAFKA_HEAP_OPTS: '-Xms256m -Xmx768m'
+      KAFKA_BROKERS: 1
+      CONNECT_LOG4J_LOGGERS: 'org.reflections=ERROR'
+
+  timescaledb:
+    image: timescale/timescaledb:latest-pg11
+    environment:
+      TIMESCALEDB_TELEMETRY: 'off'
+      POSTGRES_USER: grafana
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - '5433:5432'
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+
+  grafana:
+    image: grafana/grafana:7.1.0-beta1
+    depends_on:
+      - timescaledb
+      - radar-jdbc-connector
+    user: '1000'
+    environment:
+      GF_SECURITY_ADMIN_PASSWORD: ${GRAFANA_PASSWORD}
+      GF_SERVER_ROOT_URL: ${GRAFANA_URL}
+      GF_USERS_VIEWERS_CAN_EDIT: 'true'
+      GF_INSTALL_PLUGINS: 'flant-statusmap-panel'
+      GF_USERS_DEFAULT_THEME: light
+      GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH: '/etc/grafana/provisioning/dashboards/home.json'
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_HOST: host.docker.internal:5433
```

Comment:
Not sure about this one... Should be `timescaledb:5432`?

Reply:
Made the changes.